tests/test_ddl.py

```diff
@@ -0,0 +1,22 @@
+import sqlglot
+
+from fakesnow.transforms.ddl import alter_table_add_multiple_columns
+
+
+def test_alter_table_add_multiple_columns() -> None:
+    # Test multiple columns are split into separate statements
+    sql = "alter table tab1 add column col1 int, col2 varchar(50), col3 boolean"
+    result = alter_table_add_multiple_columns(sqlglot.parse_one(sql))
```

Comment:
this will make sure it's parsed as exp.Alter
```suggestion
    result = alter_table_add_multiple_columns(sqlglot.parse_one(sql, read="snowflake"))
```